shc/web/templates/base.htm

```diff
@@ -61,18 +61,26 @@
                 <div class="ui container">
                     {% for label, icon, link in menu %}
                         {% if link is string %}
-                            <a class="mobile hidden item" href="{{ root_url }}/page/{{ link }}/">
+                            <a class="mobile hidden item {% if '/page/' ~ link ~ '/' == current_url %}selected{% endif %}"
```

Comment:
Here, the page id should be used instead of the current url. As far as I see, it is currently not available to the template, but it can easily be added to the template data in `shc.web.interface.WebServer._page_handler()`.

Reply:
As discussed moved the logic to python.  Still need to use the url path to identify the current menu.  I get this from the `request.path` thus I do not know the page id but only the path.  Or do I miss something?

Reply:
Yes, you can simply access the `name` attribute of the `WebPage` object in the `_page_handler()` method. See my comment in `shc/web/interface.py`.

Reply:
Yep this is perfect.  But you mentioned the `page id` and I do not have this in my menu structure.  But comparing the name is unambiguous so we should be fine.